tests/test_authentication.py

```diff
@@ -0,0 +1,138 @@
+import mock
+import pytest
+import time
+
+from datetime import datetime
+from flask import url_for
+from mock import Mock, patch
+
+from app.blueprints.base.models import User, Suit
+from app.blueprints.base.views import authenticated_within
+
+max_age = 50
+
+mock_time_more_than_max_age = Mock()
+mock_time_more_than_max_age.return_value = time.mktime(
+    datetime(2011, 6, 21, 10, 10, 0).timetuple())
+
+mock_time_less_than_max_age = Mock()
+mock_time_less_than_max_age.return_value = time.mktime(
+    datetime(2011, 6, 21, 10, 0, max_age - 5).timetuple())
+
+mock_session = Mock()
+mock_session = {'iat': time.mktime(
+    datetime(2011, 6, 21, 10, 0, 0).timetuple())}
+
+mock_openid_config = Mock()
+mock_openid_config.return_value = {'authorization_endpoint':
+                                   'http://dex.example.com:5556/auth',
+                                   'discovery_url':
+                                   'http://dex.example.com:5556',
+                                   'client_id': None}
```

Comment:
Although this style of indentation is acceptable per PEP8, it is better not to have arbitrary indentation levels like this. Also, having the values indented at the same level as the keys makes this harder to read.

Prefer indenting all the elements one level, rather than indenting the second and following items to the level of the first. Eg:
```python
mock_openid_config.return_value = {
    'authorization_endpoint': 'http://dex.example.com:5556/auth',
    'discovery_url': 'http:// ...
```